services/accelerometer.md

```diff
@@ -21,15 +21,15 @@ We recommend an orientation marking on the PCB so that users can mount modules w
 
 ## Registers
 
-    ro forces @ reading {
+    volatile ro forces @ reading {
```

Comment:
aren't all readings implicitely volatile?

Reply:
That's a good question. Are they?

Reply:
Note I still need to go through all our specs to add volatile. If readings are implicitly volatile, it will certainly make this task easier.

Reply:
They are and there is already registers to specify the interval. volatile should only be allowed on rw registers

Reply:
otherwise, it's a const

Reply:
OK. Updated the code so that volatile only applies to ro/rw (and so comes after those).